install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -707,11 +707,11 @@ data:
 
 {{- if eq $ipam "cluster-pool" }}
 {{- if .Values.ipv4.enabled }}
-  cluster-pool-ipv4-cidr: {{ .Values.ipam.operator.clusterPoolIPv4PodCIDR | quote }}
+  cluster-pool-ipv4-cidr: {{ .Values.ipam.operator.clusterPoolIPv4PodCIDR | join " " | quote }}
   cluster-pool-ipv4-mask-size: {{ .Values.ipam.operator.clusterPoolIPv4MaskSize | quote  }}
 {{- end }}
 {{- if .Values.ipv6.enabled }}
-  cluster-pool-ipv6-cidr: {{ .Values.ipam.operator.clusterPoolIPv6PodCIDR | quote }}
+  cluster-pool-ipv6-cidr: {{ .Values.ipam.operator.clusterPoolIPv6PodCIDR | join " " |quote }}
```

I don't think using space works as a separator for pflag.StringSlice.

```suggestion
  cluster-pool-ipv6-cidr: {{ .Values.ipam.operator.clusterPoolIPv6PodCIDR | join "," |quote }}
```

---

https://github.com/cilium/cilium/blob/f3597588e0bbc29045a669f759cc4aa99dc34953/pkg/option/config.go#L2330

I checked this place and if "," is used, it is judged to be a whole string and not a list of strings.
Because the parameters are read from `ReadDirConfig`

---

Ah, you seem to be correct correct! Thanks for checking!